docs/server/features/streams.md

```diff
@@ -4,17 +4,81 @@ EventStoreDB is purpose-built for event storage. Unlike traditional state-based
 
 These **events** are logically organized into **streams**, typically only one stream per entity.
 
-## Metadata and reserved names
+## Types of Streams
+
+### User-Created Streams
+
+User-created streams are streams that are explicitly created by users to organize events related to their application’s entities or processes. Examples of user-created stream names include:
+
+- foo
+- order-12345
+- user-johndoe
+
+### System Streams
+
+System streams are used internally by EventStoreDB for managing its internal operations and features. System stream names always start with the $ prefix. Examples of system stream names include:
+
+- $stats
+- $projections
+- $scavenges
+
+::: warning
+EventStoreDB uses a **`$`** prefix for all internal data, such as **`$maxCount`** in a stream's metadata, or **`$all`** in a stream name. Do not use a **`$`** prefix for event names, metadata keys, or stream names.
+Doing so may result in conflicts with internally created streams. It is also not recommended to write data to or modify existing data in the system streams except as detailed below.
+:::
+
+#### Non-specific System Streams
+
+The following system streams are created by the system regardless of which subsystem is active:
 
-## Stream metadata
+| Stream name format      | Purpose                                                                                                                                                                        |
+|:------------------------|:-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **`$all`**              | Record of all the events in the database irrespective of stream. Events in the $all stream are ordered in the sequence they are written to the database.                       |
+| **`$users`**            | Record of users in the database. All events have the reserved type **`$User`**.                                                                                                |
```

Comment:
`$users` and `$users-{name}` streams will only be created if [Basic authentication](https://developers.eventstore.com/server/v24.10/security/user-authentication.html#basic-authentication) is enabled, and won't be created if using Insecure, LDAPS, or OAuth.